code/src/components/AddTask/AddTask.css

```diff
@@ -0,0 +1,51 @@
+.form-container {
+  border: 2px solid black;
+  box-shadow: 6px 6px 1px 1px #000;
+  display: flex;
+  padding: 1%;
+  margin-top: 3%;
+  margin-left: auto;
+  margin-right: auto;
+  width: 90%;
+}
+
+.input-span {
+  display: inline-block;
+  position: relative;
+  width: 90%;
+}
+
+.form-input {
+  background-color: #000;
+  border: 0.1px solid #292b2e;
+  color: #fff;
+  flex: 1;
+  padding: 0.5em 1em 0.5em 2.3em;
+  font-size: 1.1em;
+  border-radius: 12px;
+  -webkit-border-radius: 12px;
+  -moz-border-radius: 12px;
+  -ms-border-radius: 12px;
+  -o-border-radius: 12px;
+  width: 90%;
+}
+
+.form-input::placeholder {
+  font-family: 'Josefin Sans', sans-serif;
+}
+
+.task-button {
```

Comment:
Your design makes it very clear that this is a button, but it could be highlighted even more by adding "cursor: pointer", makes it easy for the user to understand that they can "press" it.

Reply:
Oh indeed! I've just tested it and it works great. Amazing how a small detail can make such a difference for the user!